exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/SchemaUtils.kt

```diff
@@ -317,8 +317,9 @@ object SchemaUtils {
      * By default, a description for each intermediate step, as well as its execution time, is logged at the INFO level.
      * This can be disabled by setting [withLogs] to `false`.
      *
-     * **Note:** Some dialects, like SQLite, do not support `ALTER TABLE ADD COLUMN` syntax completely.
-     * Please check the documentation.
+     * **Note:** Some databases, like **SQLite**, only support `ALTER TABLE ADD COLUMN` syntax in very restricted cases,
+     * which may cause unexpected behavior when adding some missing columns. Please check the documentation.
```

Comment:
'Please check the documentation.' -> 'For more information, refer to the relevant documentation.'

Reply:
👍 Changed in all places